tools/cli/src/mentat_cli/repl.rs

```diff
@@ -207,11 +207,15 @@ impl Repl {
     }
 
     pub fn explain_query(&self, query: String) {
-        let explanation = match self.store.explain_query(query) {
-            Result::Ok(vals) => vals,
+        let (query, steps) = match self.store.explain_query(query) {
```

Comment:
If you put the match outside, you can avoid the `return`s and flatten the function some:

```
match self.store.explain_query(query) {
    Result::Err(err) => println!(…),
    Result::Ok(QueryExplanation::KnownEmpty(because)) => println!(…),
    Result::Ok(QueryExplanation::ExecutionPlan { query, steps }) => {
        …
    }
```